eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala

```diff
@@ -94,11 +98,13 @@ object ChannelRelayer {
             case WrappedLocalChannelUpdate(LocalChannelUpdate(_, channelId, shortChannelId, remoteNodeId, _, channelUpdate, commitments)) =>
               context.log.debug(s"updating local channel info for channelId=$channelId shortChannelId=$shortChannelId remoteNodeId=$remoteNodeId channelUpdate={} commitments={}", channelUpdate, commitments)
               val channelUpdates1 = channelUpdates + (channelUpdate.shortChannelId -> Relayer.OutgoingChannel(remoteNodeId, channelUpdate, commitments))
-              apply(nodeParams, register, channelUpdates1, node2channels.addOne(remoteNodeId, channelUpdate.shortChannelId))
+              val node2channels1 = node2channels.addOne(remoteNodeId, channelUpdate.shortChannelId)
+              apply(nodeParams, register, channelUpdates1, node2channels1)
```

Comment:
Why did you make it an explicit variable?
Is it used differently in feature branches?

Reply:
It is more consistent than doing the operation in the function call.